RawWaterManagement/Spillway/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,146 @@
+  },
+  "id": "urn:ngsi-ld:dataModel:id:HDWO:33110928",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1993-03-09T19:34:50Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2006-06-14T21:09:34Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "State late teach word. Throw political music."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Very red could catch bit better. Defense sell mother. Agency lead character arm drop country."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Who usually similar yeah from report few."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Less foot defense mention price work anyone. Draw young my result. Senior life long system science respond such."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Person season available. Send church food. Reduce accept enjoy lead."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:EUUN:79828978",
+      "urn:ngsi-ld:dataModel:items:REVJ:43009644"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:ATVV:58241235",
+      "urn:ngsi-ld:dataModel:items:IVLM:48258577"
+    ]
+  },
+  "type": "Spillway",
+  "tag": {
+    "type": "Property",
+    "value": "Mean white admit father. Over especially sign event he sister. Hard down difference system organization accept. The half cause sure nothing fill."
+  },
+  "spillwayType": {
+    "type": "Property",
+    "value": "Broad-Crested"
+  },
+  "crestElevation": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "crestLength": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "spillwayWidth": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "numberAbutments": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "apronElevation": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "apronLength": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "dischargeCoefficient": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "designHead": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "designDischarge": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "designDischargeCoefficient": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "maxFloodElevation": {
+    "type": "Property",
+    "value": 864.6
+  },
+  "controlCrossSection": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:controlCrossSection:JXFD:60487647"
+  },
+  "curveElevationDischarge": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:curveElevationDischarge:CBWI:21948924"
+  },
+  "curveDischargeCoefficient": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:curveDischargeCoefficient:MWGU:81565938"
+  },
+  "curveDesignDischargeCoefficient": {
```

Comment:
type should be Relationship
The proper way to model Relationships is as follow :

"observedBy":{
	"type":"Relationship",
	"object":" urn:ngsi-ld:Device:01 "	}